zksync_sdk/zksync_provider/v01.py

```diff
@@ -48,6 +49,13 @@ async def submit_txs_batch(self, transactions: List[TransactionWithSignature],
         trans_ids: List[str] = await self.provider.request("submit_txs_batch", [transactions, signatures])
         return [Transaction.build_transaction(self, trans_id) for trans_id in trans_ids]
 
+    async def submit_trx_batch_v2(self, transactions: List[TransactionWithOptionalSignature],
```

Comment:
> submit_trx_batch_v2

`submit_batch_builder_txs_batch` ? 

the `v2` doesn't tell the user how it is different from the `submit_txs_batch`.

Reply:
Done